src/js/component.js

```diff
@@ -1239,9 +1239,7 @@ class Component {
     fn = Fn.bind(this, fn);
 
     const timeoutId = window.setTimeout(fn, timeout);
```

Comment:
I almost commented saying that this should be changed to `this.setTimeout` before realizing what this line is for 😆